modules/ROOT/pages/object-store/object-store-to-watermark.adoc

```diff
@@ -71,4 +71,4 @@ The last step (in `<os:store/>`) is to update the watermark value:
 Note that the repeatable streaming feature prevents the need for you to know the integration logic. Even though the query operation auto-paged, this feature guarantees that you can still iterate the result set again.
 +
 * `failIfPresent` is set to `false` so that the value will be updated if it is present. Otherwise, the operation would only work the first time and fail if the watermark were already in the object store.
-* `failOnNullValue` is set to `false`, so if the query comes back empty, the DataWeave expression (which looks for the max value) will return null. Setting it to `true` makes the connector skip null values, which prevents the need to use a `<choice/>` router to check the condition.
\ No newline at end of file
+* `failOnNullValue` is set to `false`, so if the query comes back empty, the DataWeave expression (which looks for the max value) will return null. Setting it to `false` makes the connector skip null values, which prevents the need to use a <choice/> router to check the condition, while using the default value of `true` makes the connector throw a `NULL_VALUE` error.
```

Comment:
Please also fix this value in the reference at https://docs.mulesoft.com/connectors/object-store/object-store-connector-reference#store